components/sections/features/variant_f.tsx

```diff
@@ -12,105 +16,93 @@ function VariantF({
   primaryButton,
 }: FeaturesProps) {
   return (
-    <section>
-      <div className="radius-for-skewed bg-gray-50 py-20">
-        <div className="container mx-auto px-4">
-          <div className="-mx-4 flex flex-wrap items-center">
-            <div className="mb-12 flex w-full px-4 lg:mb-0 lg:w-1/2">
-              <div className="max-w-md">
-                <span className="font-bold text-webriq-darkblue">
-                  {caption}
-                </span>
-                {title && (
-                  <h1 className="font-heading mb-6 text-4xl font-bold lg:text-5xl">
-                    {title}
-                  </h1>
-                )}
-                {description && (
-                  <div className="mb-6 max-w-sm">
-                    <p className="leading-loose text-gray-500">{description}</p>
-                  </div>
-                )}
-                {primaryButton?.label && (
-                  <div className="flex flex-wrap lg:-ml-5">
-                    <ConditionalLink
-                      link={primaryButton}
-                      className="lg:w-auto py-2 px-6 leading-loose lg:ml-5 text-gray-50 font-bold bg-webriq-darkblue hover:bg-webriq-blue transition duration-200 rounded-l-xl rounded-t-xl"
-                      ariaLabel={primaryButton?.label}
-                    >
-                      {primaryButton?.label}
-                    </ConditionalLink>
-                  </div>
-                )}
+    <section className="py-20 bg-gray-50">
+      <Container>
+        <Flex wrap align="center">
+          <Flex direction="col" className="w-full mb-12 lg:mb-0 lg:w-1/2">
+            <Container maxWidth={448} className="ml-0">
+              <Text weight="bold" className="font-bold text-primary">
```

Comment:
Kindly conditionally render the Caption element to only display if it is defined/has value.